package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-domain-manager",
-  "version": "3.2.6",
+  "version": "3.3.0",
```

Comment:
Definitely a major version bump, this is a `4.0.0` change. This is a huge lift, thanks for undertaking it.